manifests/mod/worker.pp

```diff
@@ -40,6 +41,10 @@
 # @param apache_version
 #   Used to verify that the Apache version you have requested is compatible with the module.
 #
+# @param maxrequestworkers
+#   Maximum number of connections that will be processed simultaneously
```

Comment:
I think this will show up slightly better in the rendered strings documentation
```suggestion
#   Maximum number of connections that will be processed simultaneously.
```